data/devportal_topic_icons/people.svg

```diff
@@ -1,9 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<svg width="60px" height="60px" viewBox="0 0 60 60" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
-    <!-- Generator: Sketch 63.1 (92452) - https://sketch.com -->
-    <title>people</title>
-    <desc>Created with Sketch.</desc>
-    <g id="people" stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
-        <path d="M26.238,38.128 C30.824,37.068 35.092,36.14 33.026,32.236 C26.732,20.354 31.356,14 38.002,14 C44.778,14 49.288,20.598 42.978,32.238 C40.848,36.166 45.276,37.092 49.764,38.13 C53.734,39.046 54,40.986 54,44.344 L53.994,46 L51.336,46 C51.336,41.822 51.502,41.266 48.884,40.662 C45.082,39.786 41.494,38.958 40.182,36.054 C39.704,34.994 39.392,33.25 40.634,30.968 C43.378,25.904 44.072,21.516 42.532,18.934 C40.728,15.9 35.298,15.916 33.508,18.89 C31.972,21.436 32.656,25.848 35.38,30.99 C36.594,33.282 36.274,35.022 35.792,36.076 C34.472,38.966 30.848,39.802 27.012,40.686 C24.508,41.266 24.668,41.862 24.668,46 L22.006,46 C22.006,41.608 21.654,39.188 26.238,38.128 L26.238,38.128 Z M6.004,46 L8.662,46 C8.662,42.164 8.29,43.23 12.31,42.054 C14.338,41.464 16.13,40.608 16.942,38.83 C17.366,37.904 17.652,36.39 16.618,34.436 C14.714,30.84 14.18,27.688 15.194,26.006 C16.288,24.188 19.734,24.19 20.832,26.036 C22.702,29.17 19.246,34 18.792,36 L21.584,36 C22.464,34 23.996,31.584 23.996,28.2 C23.996,24.18 21.372,22 18,22 C13.014,22 9.546,26.766 14.268,35.678 C15.816,38.606 12.616,39.302 9.178,40.096 C6.198,40.786 6,42.24 6,44.764 L6.004,46 Z" stroke="#000000" fill="#000000" fill-rule="nonzero"></path>
-    </g>
-</svg>
\ No newline at end of file
+<!DOCTYPE svg PUBLIC "-//W3C//DTD SVG 1.1//EN" "http://www.w3.org/Graphics/SVG/1.1/DTD/svg11.dtd">
```

Comment:
Because the file is only ever loaded and not inlined, I think the straggling doctype here is OK. HTML5 validator is happy